src/app/main/component/comments/components/like-comment/like-comment.component.ts

```diff
@@ -42,13 +42,14 @@ export class LikeCommentComponent implements OnInit {
   }
 
   checkSocketMessageToSubscribe() {
-    if (this.router.url.includes('news')) {
+    const baseUrl = '/app/likeAndCount';
```

Comment:
if a constant is only used once, it's more concise to use it directly rather than declaring it as a constant